eu.betterbird.Betterbird.json

```diff
@@ -1,70 +1,298 @@
 {
-    "app-id": "eu.betterbird.Betterbird",
-    "runtime": "org.freedesktop.Platform",
-    "runtime-version": "22.08",
-    "sdk": "org.freedesktop.Sdk",
-    "command": "betterbird",
-    "rename-icon": "betterbird",
-    "finish-args": [
-        "--share=ipc", "--socket=x11",
-        "--device=dri",
-        "--share=network",
-        "--socket=pulseaudio",
-        "--persist=.thunderbird",
-        "--filesystem=xdg-download:rw",
-        "--talk-name=org.a11y.Bus",
-        "--talk-name=org.freedesktop.Notifications",
-        "--socket=pcsc",
-        "--require-version=0.10.3",
-        "--filesystem=~/.gnupg",
-        "--filesystem=xdg-run/gnupg:ro",
-        "--filesystem=/run/.heim_org.h5l.kcm-socket",
-        "--env=GTK_PATH=/app/lib/gtkmodules",
-        "--socket=cups"
-    ],
-    "cleanup": [
-        "/include",
-        "/lib/pkgconfig",
-        "/lib/girepository-1.0",
-        "/share/pkgconfig",
-        "/share/aclocal",
-        "/share/gtk-doc",
-        "/share/doc",
-        "/share/info",
-        "/share/gir-1.0",
-        "/share/man",
-        "/man",
-        "*.la", "*.a"
-    ],
-    "modules": [
-        "shared-modules/dbus-glib/dbus-glib.json",
-        "shared-modules/libcanberra/libcanberra.json",
-        {
-            "name": "betterbird",
-            "buildsystem": "simple",
-            "build-commands": [
-                "install -d /app/lib/betterbird",
-                "cp -r * /app/lib/betterbird/",
-                "ln -s /app/lib/betterbird/betterbird /app/bin/betterbird",
-                "for i in 16 22 24 32 48 64 128 256;do install -Dm644 chrome/icons/default/default${i}.png /app/share/icons/hicolor/${i}x${i}/apps/betterbird.png;done",
-                "install -Dm644 eu.betterbird.Betterbird.desktop /app/share/applications/eu.betterbird.Betterbird.desktop",
-                "install -Dm644 eu.betterbird.Betterbird.appdata.xml /app/share/metainfo/eu.betterbird.Betterbird.appdata.xml"
-            ],
-            "sources": [
-                {
-                    "type": "archive",
-                    "url": "https://www.betterbird.eu/downloads/LinuxArchive/betterbird-102.2.2-bb16.en-US.linux-x86_64.tar.bz2",
-                    "sha256": "ef2f526b934261e7d57670e672b304b84ff76c215512891c97adfef90d53bb13"
-                },
-                {
-                    "type": "file",
-                    "path": "eu.betterbird.Betterbird.desktop"
-                },
-                {
-                    "type": "file",
-                    "path": "eu.betterbird.Betterbird.appdata.xml"
-                }
-            ]
+  "id": "eu.betterbird.Betterbird",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "21.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable",
+    "org.freedesktop.Sdk.Extension.node14",
```

Comment:
any chance to move to node16? It seems that node14 doesn't build with the 22.08 runtime.

not blocking at the moment.

Reply:
Frankly I have no idea where Thunderbird/Betterbird relies on node.js and if that part will be compatible with node16. Maybe @Betterbird can comment on that?

I don't really see any downside of staying with freedesktop 21.08 for now, so the pressure for upgrading to node16 is rather low I'd say.

Reply:
Taking into account Betterbird's comment on node.js being part of the Mozilla platform I think it is out of scope of this flatpak to test if Thunderbird/Betterbird works with node16. So I will keep the runtime 21.08 and node14 until the Thunderbird flatpak is upgraded.